frontend/app/components/user-component/my-badges.js

```diff
@@ -91,7 +90,14 @@ export default Component.extend({
       }
     },
     batchdownload() {
-      var arr = $('.checkbox');
+      var arr = $('.checkbox:checkbox:checked');
+      if (arr.length === 0) {
+        this.get('notifications').error('Please select Badges to Download!', {
```

Comment:
`Please select at least one badge to download` would be a better statement, I suppose